openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash -eux
+#
+# Run Charmed Openstack CI tests manually in a similar way to how they are run
+# by OpenStack CI (OSCI).
+#
+# Usage: clone/fetch charm to test and run from within charm root dir.
+#
+FUNC_TEST_PR=
+FUNC_TEST_TARGET=
+SKIP_BUILD=false
+
+usage () { echo "USAGE: --func-test-target | --func-test-pr | --skip-build"; }
+
+while (($# > 0))
```

Comment:
If you call the script without any arguments and you're not inside a charm directory it eventually fails. It should check for the current directory being a charm somehow.

Reply:
i will fix this as a followup patch but have put info in the help message about how to run the tool